src/charm.py

```diff
@@ -87,14 +92,14 @@ def __init__(self, *args):
             ports=[
                 ServicePort(name="f1c", port=38472, protocol="SCTP"),
                 ServicePort(name="f1u", port=self._charm_config.f1_port, protocol="UDP"),
-                ServicePort(name="rfsim", port=4043, protocol="TCP"),
+                ServicePort(name="rfsim", port=RFSIM_PORT, protocol="TCP"),
```

Comment:
It seems some of the changes here conflict with changes in #24 let's make sure you two align on the right approach

Reply:
Service definition for RFSIM is not required. This line is removed.